base_layer/p2p/src/services/liveness/state.rs

```diff
@@ -181,6 +193,34 @@ impl LivenessState {
             .get(node_id)
             .and_then(|latency| Some(latency.calc_average()))
     }
+
+    pub fn add_node_id(&mut self, node_id: &NodeId) -> Result<(), LivenessError> {
+        if self.nodes_to_monitor.contains_key(node_id) {
```

Comment:
Think a Result return type is not necessary here - it does not matter if the same node id is added twice

Reply:
done